Update from 2.14.2 -> 2.15

fultonesser opened the pull request:
Updating dev branch to 2.15 so that we can test if this improves some of the issues we've been encountering

fultonesser committed:
Update from 2.14.2 -> 2.15

dragon-dxw commented on app/poetry.lock:
```diff
@@ -1585,11 +1585,11 @@ test = ["pytest", "pytest-cov"]
 [[package]]
 name = "xlsxwriter"
-version = "1.4.5"
+version = "3.0.2"
```
That's a big version boost. Should be fine, generally in favour but a candidate if there's any problems... with writing Excel spreadsheets, which I don't think anything does?

dragon-dxw commented:
https://github.com/nhsx/nhsx-website/blob/1615b0a5bebdb2334d76902c51b921cb97e10e14/app/config/settings/test.py#L32 and https://github.com/nhsx/nhsx-website/search?q=wagtail.contrib.postgres might be worth updating in line with the deprecation notice
https://docs.wagtail.io/en/latest/releases/2.15.html#database-search-backends-replaced, but might well be worth taking a second pass at.

fultonesser committed:
Updating Backend search config

- updated as specified in version notice, https://docs.wagtail.io/en/stable/releases/2.15.html#id2

- removed `wagtail.contrib.postgres_search` from base.py

fultonesser commented:
Thanks @dragon-dxw 

I've carried out those edits as suggested. Do you think there's anything else to be aware of with this version bump?
